steps/examples/step-base/lib/angular/base.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+function initModule() {
+  var moduleName = 'wfm.base';
+  var ngModule = angular.module(moduleName, []);
+
+  require('../../dist');
+
+  ngModule.directive('baseReport', function($templateCache) {
```

Comment:
BaseReport ? IMHO I think we may just say userStep etc. to make sure that people will get what we doing here without knowledge of angular. I see that average user may actual do not know what name to put here.